src/sqlfluff/core/linter/linter.py

```diff
@@ -714,23 +723,34 @@ def render_string(
                     "details."
                 )
             )
+
+        variant_limit = config.get("render_variant_limit")
+        templated_variants: List[TemplatedFile] = []
+        templater_violations: List[SQLTemplaterError] = []
+
         try:
-            templated_file, templater_violations = self.templater.process(
+            for variant, vs in self.templater.process_with_variants(
```

Comment:
can we have a more descriptive variable name than `vs`?

Reply:
yeah - good point